docs/guides/intro-to-graphql.md

```diff
@@ -170,15 +170,17 @@ query GetUsers {
   }
 }`;
 
-fetch('/admin/api', {
-  method: 'POST',
+fetch("/admin/api", {
+  method: "POST",
   headers: {
-    'Content-Type': 'application/json',
+    "Content-Type": "application/json",
   },
-  body: {
+  body: JSON.stringify({
     query: GET_ALL_USERS,
-  },
-}).then(result => result.json());
+  }),
+})
+  .then((result) => result.json())
+  .then((r) => updateData(r));
```

Comment:
Is `updateData` and example of how to do something with the result? If so, maybe we should just assign it to something? i.e?

`const result = fetch(...)`